pages/features.html

```diff
@@ -6,92 +6,122 @@
     </p>
 
     <center><img src="{% static '/static/images/feature_sampler2.png' %}"></center>
-
-    <H2>{% trans "Built-in Effects" %}</H2>
+    
+    <h2>{% trans "Built-in Effects" %}</h2>
     <p>
-      {% trans "Add sizzle to your mix with reverb, echo, flange, and more." %}
+      {% trans "Use reverb, echo, flange, and more effects to blend music in your own signature ways." %} <!--TODO: Say a little more-->
     </p>
 
-    <div style="clear: both;"></div>
+  <div style="clear: both;"></div>
+  
+    <h1>{% trans "Customizable DJ Hardware Support" %}</h1>
+    {% trans "Free as in freedom as well as price, Mixxx plays nice with a variety of hardware without artificial restrictions or vendor lock-in mechanisms. The free version is the full version. Pick and choose gear from any source to build your unique setup." %}<!--TODO: Alignment somehow is messed up here.-->
+    
+    <div class="halfbox_left">
+      <div style="height: 200px;">
+        <center><img width="340px" src="{% static '/static/images/feature_midi2.png' %}" style="padding-top: 20px;"></center>
+      </div>
+
+      <h2>{% trans "MIDI and HID Controller Support" %}</h2>
+      <p>
+        {% blocktrans with hardware_compatibility_wiki_link="/wiki/doku.php/hardware_compatibility" %}<b>Included Presets</b><br> for controllers such as the Traktor Kontrol S4 MK2, Pioneer DDJ-SB &amp; DDJ-SB2, Allen &amp; Heath Xone K2, Hercules DJ Console Series, and many more. <a href="{{ hardware_compatibility_wiki_link }}">Full list of supported devices</a>{% endblocktrans %}
+      </p>
 
-    <H1>{% trans "Powerful Library" %}</H1>
+      <p><b>{% trans "Programmable Mapping Engine" %}</b><br>
+        {% blocktrans with midi_scripting_wiki_link="/wiki/doku.php/midi_scripting" %}Cutting-edge JavaScript-based scripting engine provides maximum flexibility. MIDI or HID messages can be mapped onto custom functions which execute complex behaviour. <a href="{{ midi_scripting_wiki_link }}">Read more</a>{% endblocktrans %}
+      </p>
+    </div>
+    <div class="halfbox_right">
+      <div style="height: 200px;">
+        <center><img src="{% static '/static/images/feature_vinylcontrol.png' %}"></center>
+      </div>
+
+      <h2>{% trans "Free Timecode Vinyl Control" %}</h2>
+      <p>
+        {% trans "Control your digital music files from turntables or CDJs and a mixer using timecoded vinyl or CDs. Mixxx is the only free timecode vinyl control software for Windows, Mac OS X, and Linux." %}
+      </p>
+
+      <p>
+        {% blocktrans with vinyl_control_manual_link="/manual/latest/chapters/vinyl_control.html" %}Serato CV 2, Serato CV 2.5, Traktor Scratch Mk1, and Mixvibes DVS V2 timecode are supported with absolute positioning, needle cueing, and end-of-record detection. Use any sound card(s) your OS supports to read timecode signals and play your music in high quality. <a href="{{ vinyl_control_manual_link }}">Read more</a>{% endblocktrans %}
```

Comment:
Good that you link to the manual instead of the wiki here. It provides much more complete information.